plat/mediatek/common/mtk_sip_svc.h

```diff
@@ -62,5 +100,6 @@ enum {
  * Return MTK_SIP_E_SUCCESS on success, and MTK_SIP_E_INVALID_PARAM on failure.
  */
 uint64_t mt_sip_set_authorized_sreg(uint32_t sreg, uint32_t val);
+void save_kernel_info(uint64_t pc, uint64_t r0, uint64_t r1, uint64_t k32_64);
```

Comment:
Minor: Seems odd to declare this in a common header then define it in the platform specific code.